### PR TITLE
Feat/form creation

### PR DESCRIPTION
**Pull request by MihlaliMgwebi**
Create form for itinerary creation
-[X] UI component library
-[X] Validation
-[X] Posting Data to DB

### CODE REVIEW

File: src/app/components/currency/currency.component.ts [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
-  }
-  get f(){
-    return this.form.controls;
-  }
-
-  submit(){
-    console.log(this.form.value);
-  }
-
+  //
```
Comment (MihlaliMgwebi) [inline src/app/components/currency/currency.component.ts:23]: WIll remove before merge

File: src/app/components/auth/auth.component.html [hunk 2]
```diff
@@ -1,10 +1,55 @@
-<form>
-  <input type="email" #email>
-  <input type="password" #password>
+<!--  <button *ngIf="isLoggedIn$ | async" type="button" (click)="signOut()">Sign Out</button>-->
```
Comment (CharelMarais) [inline src/app/components/auth/auth.component.html:1]: Remember to remove if wont be implemented

File: src/app/components/page-not-found/page-not-found.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<p>page-not-found works!</p>
```
Comment (CharelMarais) [inline src/app/components/page-not-found/page-not-found.component.html:1]: Yes, it does.   XD

**Review by CharelMarais — APPROVED** [review]
There are a few commented code snippets in the code. 

If you are using 2 separate collections for trips and the itinerary, what happens to the related itinerary items if you delete the trip?

No issues found in the code it makes sense approved!

**Pull request merged**